packages/code-du-travail-frontend/src/search/Search.js

```diff
@@ -142,12 +156,32 @@ class Search extends React.Component {
   };
 
   onSearch = ({ value }) => {
-    const { excludeSources } = this.state;
-    suggestResults(value, excludeSources).then(results => {
-      this.setState({ suggestions: results.hits.hits }, () => {
-        this.props.onResults(results);
+    const { facet, excludeSources } = this.state;
+    const promise =
```

Comment:
`promise` is not a very explicite variable name...
it's like 
- `const number = 42`
- `const string = "foo"`
- `const boolean = true`
- `const object = {}`
- `const array = []`
- `const fn = () => null`

It's an async search result ?

Reply:
🌮